src/Buffer.ts

```diff
@@ -34,6 +34,9 @@ export default class Buffer extends events.EventEmitter {
         var current = {attributes: <i.Attributes>null, text: ''};
         var cursorPosition = cursor.getPosition();
 
+        /* Quick fix for the strange character at the end (https://github.com/black-screen/black-screen/pull/56#issuecomment-139653723) */
+        if (process.platform === 'win32' && row.length < 1) return;
```

Comment:
Triple equal is useless in TypeScript. In JS it compares with types, but, you know, **Type**Script compares types out of the box. :)